test/descriptive.test.js

```diff
@@ -30,6 +31,26 @@ describe('Descriptive Statistics', () => {
     testUndefinedWithNullable(max);
   });
 
+  test('Sum', () => {
+    expect(sum([])).toBe(0);
+    expect(sum(3)).toBeUndefined();
+    expect(sum([3])).toBe(3);
+    expect(sum([1, 2, 3, 4, 5])).toBe(15);
+    expect(sum([0.1, 0.2, 0.3, 0.4, 0.5, 0.6, 0.7, 0.8, 0.9,1.0, 1.1, 1.2, 1.3, 1.4, 1.5, 1.6, 1.7]))
+      .toBe(15.3);
+
+    var veryLongArray = [10000, 0];
+    for (let i = 1; i < 501; i++) {
+      veryLongArray[i] = 0.000001;
+    }
+    expect(sum(veryLongArray)).toBe(10000.0005);
```

Comment:
This is not necessary I think. Could you please explain the reasoning behind this ?

Reply:
Hi! I wanted to ensure the error accumulation was working properly but your example of sum(0.1 through 1.7 step 0.1) demonstrates it well enough compared to the naive algorithm. Have removed.